src/OpenTelemetry.Abstractions/Context/Propagation/TraceContextFormat.cs

```diff
@@ -54,7 +54,7 @@ public SpanContext Extract<T>(T carrier, Func<T, string, IEnumerable<string>> ge
                     return SpanContext.Blank;
                 }
 
-                var traceparent = traceparentCollection.First();
+                var traceparent = traceparentCollection?.First();
```

Comment:
I don't understand why this change is in this PR?

Reply:
i think it's an accidental bugfix